src/server/utils/stateConfig.js

```diff
@@ -79,7 +79,7 @@ if (process.env.NODE_ENV === 'development' && fs.existsSync(pathToDevEndpoints))
   Object.entries(devEndpoints).forEach(([configName, { devProxyPath }]) => {
     const value = url.format({
       protocol: 'http',
-      hostname: ipAddress,
+      hostname: 'localhost',
```

Comment:
If we are going to remove `ipAddress` can you also remove https://github.com/americanexpress/one-app/blob/master/src/server/utils/stateConfig.js#L75?

Reply:
Also the import please:

https://github.com/americanexpress/one-app/blob/master/src/server/utils/stateConfig.js#L17